Blueprints/using-powershell-in-packages-and-scripts.md

```diff
@@ -0,0 +1,24 @@
+{{{
+  "title": "Using PowerShell in Packages and Scripts",
+  "date": "5-31-2015",
+  "author": "Ted Henley",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Related
+* [Understanding the Difference Between Templates, Blueprints and Packages](understanding-the-difference-between-templates-blueprints-and-packages.md)
```

Comment:
all links to other articles need to have the full path per the readme
https://github.com/CenturyLinkCloud/PublicKB/blob/master/README.md#links-kb-article-to-kb-article

i.e. '(../Blueprints/KB.md)'

this changed a while back so you'll need to update all the links in this kb